.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,13 +27,20 @@ Optionally add one or more of the following kinds if applicable:
 
 #### What this PR does / why we need it:
 
-#### Which issue(s) this PR fixes:
+#### Which issue(s) this PR is related to:
```

Comment:
I would also suggest to put some wording on how to link KEPs. Unfortunately linking KEPs via Additonal Context is not creating a link as it is placed as a markdown text.